rhobs/alerting/data_plane/prometheus.stability_image_repository_provision_failures_alerts.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-stability-image-repository-provision-failures-alerting-rules
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: stability-image-repository-provision-failures
+    interval: 1m
+    rules:
+    - alert: StabilityImageRepositoryProvisionFailures
+      expr: |
+        (redhat_appstudio_imagecontroller_image_repository_provision_failure_time_count[30m]) > 5
```

Comment:
Have a look at the query functions on the link below. You want to see if the change over the last 30 minutes was larger than 5. Have a look at the `rate` function. It might be what you're after.
https://prometheus.io/docs/prometheus/latest/querying/functions/#rate

Reply:
also I don't want any rate, I want to have alert when MORE than 5 (really 5 in 30 minutes) metrics occur in last 30 minutes

Reply:
`increase` should do the trick:
https://prometheus.io/docs/prometheus/latest/querying/functions/#increase

So, assuming there's only ever one unique time series for this metric:
```
increase(redhat_appstudio_imagecontroller_image_repository_provision_failure_time_count[30m]) > 5
```

Otherwise you may want to sum them up:
```
sum(increase(redhat_appstudio_imagecontroller_image_repository_provision_failure_time_count[30m])) > 5
```